### PR TITLE
Testable automate uptime checks

### PR DESCRIPTION
**Pull request by njseeto**
[Trello card](https://trello.com/c/K1ZZo2Uj/2033-automate-pingdom)

## Context

Instead of using the worker queues to automate the creation and deletion
of Uptime Checks we are going to use an admin dashboard page so a
developer can create and destroy them as required.

This page is only available in Live. Unpublishing is a manual process
anyway so we can add the destroying of an Uptime Check to the
unpublishing manual workflow.

Co-authored-by: Tomas Destefi <tomas.destefi@digital.justice.gov.uk>
Co-authored-by: Natalie Seeto <natalie.seeto@digital.justice.gov.uk>

### CODE REVIEW

File: app/services/publisher/pingdom_eligibility.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+class Publisher
+  class PingdomEligibility
```
Comment (brenetic) [inline app/services/publisher/pingdom_eligibility.rb:2]: ### class Uptime::Eligibility

just coz we don't necessarily have to make it specific to pingdom

Comment (brenetic) [reply app/services/publisher/pingdom_eligibility.rb:2]: great class name by the way 😆

**Comment by brenetic** [comment]
can we have a PR description etc please

**Review by tomas-stefano — APPROVED** [review]
🚀

**Pull request merged**